test/integration/statetest/CMakeLists.txt

```diff
@@ -143,9 +143,30 @@ set_tests_properties(
 add_test(
     NAME ${PREFIX}/execute_exported_tests
     # TODO: Broken exported tests are filtered out.
-    COMMAND evmone-statetest ${EXPORT_DIR} --gtest_filter=-*block.*:*tx.tx_non_existing_sender
+    COMMAND evmone-statetest ${EXPORT_STATE_TESTS_DIR} --gtest_filter=-*block.*:*tx.tx_non_existing_sender
```

Comment:
This will also export `eof_validation` suite. You can use it with `evmone-eoftest` test or filter it out.